python/cudf_polars/cudf_polars/dsl/expressions/selection.py

```diff
@@ -72,7 +72,7 @@ class Filter(Expr):
     def __init__(self, dtype: plc.DataType, values: Expr, indices: Expr):
         self.dtype = dtype
         self.children = (values, indices)
-        self.is_pointwise = True
+        self.is_pointwise = False
```

Comment:
Is a `Filter` not pointwise if it's children are pointwise?

Reply:
No, because the output is not the same size as the input.

Reply:
We are using `Expr.is_pointwise` to determine if the operation is "embarrassingly parallel". If not, we will need to add special lowering logic to `Select`.

Reply:
So filter operates row-wise, but it's not pointwise in the sense that the output is not the same size as the input